Stock.cc

```diff
@@ -1,24 +1,25 @@
 #include <iostream>
+#include<ctime>
+#include<cstdlib>
 #include "Stock.h"
 #include "json.hpp"
 
 using json = nlohmann::json;
 using namespace std;
 
-Stock::Stock() {
+Stock::Stock() {}
 
+void Stock::buyShares(int numSharesPurchased) {
```

Comment:
Can you add a check here as follows: 
`if numShares - numSharesPurchased < 0: return`

Reply:
Was planning to implement when we do Exception handling, but I'll do it now. I'll do it in a few other places based on our Google Docs: https://docs.google.com/document/d/1TNxrujVWFsPoGNavyXuWZ81kDV3SSv8K6_it_Nek0AI/edit

Reply:
Sounds goof